app/forms/openid_connect_authorize_form.rb

```diff
@@ -123,9 +127,9 @@ def parse_to_values(param_value, possible_values)
     param_value.split(' ').compact & possible_values
   end
 
-  def validate_acr_values
-    return if acr_values.present?
-    errors.add(:acr_values, t('openid_connect.authorization.errors.no_valid_acr_values'))
+  def validate_ial_values
+    return if ial_values.present?
```

Comment:
can we change the names back to `acr_values`? that's the parameter in the OIDC spec

Reply:
`acr_values` is still the name of the param. `ial_values` filters out the ial contexts from the `acr_values`. The validation used to check that `acr_values.present?` which worked since the only values we expected were ial contexts. Now that we may optionally receive aal contexts as well, I changed the validation to ensure that an ial con text was received.

Reply:
The error messages still say `ial_values` which is misleading IMO because API clients pass in `acr_values` --- I know you're OOO today so I'm going to push a commit to flip that back because I feel medium strongly about this

Reply:
Gotcha, yeah I didn’t grok fully before, but you’re right!